src/threadpool.c

```diff
@@ -27,6 +27,10 @@
 
 #include <stdlib.h>
 
+#include <sys/types.h>
+#include <sys/syscall.h>
+#include <unistd.h>
```

Comment:
Just curious, Windows doesn’t usually have these, right? Does it still work?

Reply:
apologies, remnants of debug code, now removed.